layouts/default.vue

```diff
@@ -152,6 +177,12 @@
         return this.fireworks
           && this.fireworks.display;
       },
+
+      mobile () {
+        return this.mounted
+          ? this.$vuetify.breakpoint.smAndDown
+          : !this.$device.isDesktopOrTablet;
+      },
```

Comment:
Is `mobile` even used in this component? I don't see any references to it in the PR